test/node/bigint.test.ts

```diff
@@ -1,89 +1,124 @@
 import { BSON } from '../register-bson';
 import { bufferFromHexArray } from './tools/utils';
 import { BSON_DATA_LONG } from '../../src/constants';
+import { BSONDataView } from '../../src/utils/byte_utils'
 
-describe('BSON BigInt serialization Support', function () {
-  // Index for the data type byte of a BSON document with a single element
-  const DATA_TYPE_OFFSET = 4;
+describe('BSON BigInt serialization Support', function() {
+  // Index for the data type byte of a BSON document with a 
+  // NOTE: These offsets only apply for documents with the shape {a : <n>}
+  // where n is a BigInt
+  class SerializedDocParts {
+    dataType: number;
+    key: string;
+    value: BigInt;
+  }
 
-  it('Serializes bigints with the correct BSON type', function () {
+  // NOTE: this function operates on serialized BSON documents with the shape { <key> : <n>}
+  // where n is some int64
+  // This function assumes that keys are properly encoded with the necessary null byte at the
+  // end and only at the end of the key
+  function getSerializedDocParts(serializedDoc: Uint8Array): SerializedDocParts {
```

Comment:
```suggestion
  /** 
   * NOTE: this function operates on serialized BSON documents with the shape { <key> : <n>}
   * where n is some int64
   * This function assumes that keys are properly encoded with the necessary null byte at the
   * end and only at the end of the key
   */
  function getSerializedDocParts(serializedDoc: Uint8Array): SerializedDocParts {
```

Makes the comment visible in editor hovers